bip-musig2.mediawiki

```diff
@@ -313,17 +314,25 @@ Input:
 ==== Nonce Generation ====
 
 Input:
-* The secret signing key ''sk'': a 32-byte array or 0-byte array (optional argument)
-* The aggregate public key ''aggpk'': a 32-byte array or 0-byte array (optional argument)
-* The message ''m'': a 32-byte array or 0-byte array (optional argument)
+* The secret signing key ''sk'': a 32-byte array (optional argument)
+* The aggregate public key ''aggpk'': a 32-byte array (optional argument)
+* The message ''m'': a byte array with ''0 &le; len(m) &le; 2<sup>64</sup>-1'' (optional argument) <ref name="mlen">In theory, the allowed message size is smaller because SHA256 accepts byte strings only up to size of 2^56-1 bytes</ref>
```

Comment:
Footnote copied from https://github.com/sipa/bips/pull/221/

```suggestion
* The message ''m'': a byte array (optional argument) <ref name="mlen">In theory, the message size is restricted due to the fact that SHA256 accepts byte strings only up to size of 2^61-1 bytes.</ref>
```

Reply:
Maybe it should also say "restricted due to the 8-byte length encoding"?

Reply:
Right, good to document that as well.